website/docs/r/codebuild_source_credential.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "aws"
+page_title: "AWS: aws_codebuild_source_credential"
+sidebar_current: "docs-aws-resource-codebuild-source-credential"
+description: |-
+  Provides a CodeBuild Source Credential resource.
+---
+
+# aws_codebuild_project
```

Comment:
Copy pasta 🍝 and sitewide documentation update that occurred after this PR was submitted 👍 

```suggestion
# Resource: aws_codebuild_source_credential
```

Reply:
fixed in ca65197